calico-cloud_versioned_sidebars/version-3.15-sidebars.json

```
@@ -344,7 +344,6 @@
                 "visibility/elastic/l7/datatypes"
               ]
             },
```

```suggestion
            }
```
I suspect that removing this comma will fix your build error.